pint.json

```diff
@@ -6,6 +6,10 @@
         "new_with_braces": {
             "anonymous_class": false,
             "named_class": false
-        }
+        },
+        "ordered_imports": {
+            "sort_algorithm": "length"
```

Comment:
This rule sorts our imports on length. No big deal here but I prefer having imports aligned based on their length.